Ports/wireguard-tools/patches/replace-strsep-with-strtok.patch

```diff
@@ -0,0 +1,28 @@
+--- wireguard-tools-1.0.20210914/src/ipc-uapi.h	2022-01-10 19:32:48.333858269 +0100
++++ wireguard-tools-1.0.20210914/src/ipc-uapi.h	2022-01-10 19:34:25.996423462 +0100
+@@ -246,7 +246,7 @@
+ 			peer->flags |= WGPEER_HAS_PERSISTENT_KEEPALIVE_INTERVAL;
+ 		} else if (peer && !strcmp(key, "allowed_ip")) {
+ 			struct wgallowedip *new_allowedip;
+-			char *end, *mask = value, *ip = strsep(&mask, "/");
++			char *end, *mask = value, *ip = strtok(mask, "/");
```

Comment:
Should we just implement `strsep(..)`? It's tiny, but I guess we don't have any other patches stubbing it out.

Reply:
I actually looked into it, but it's not POSIX so I wasn't sure. Gonna make another PR for that :+1: